docs/general/Mainnet/high_level_architecture.md

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 1
+---
+
+# High-level architecture
+
+![Architecture](./images/architecture.png)
+
+!!! . The node also gets informed from FFS about the finality status, hence the arrow should also go back.
```

Comment:
@apenzk How do you want to manage these comments?  I suppose these need to be cleared up

Reply:
yes, they should be managed by answering them before commiting the PR to main :)

Reply:
Probably not the easiest way to manage this, see comments on PR wrt this